### PR TITLE
Updating the project to work with the npm build

### PR DESCRIPTION
**Pull request by engineersamuel**
One of the core issues with the npm build is the `exr-wrap.wasm` is loaded within the `exr-wrap.js` as an XMLHttpRequest which uses the `location.href` which, in your own React app, resolves to your own React app.  That is unavoidable, the way it's currently written.  There are other issues with the modules and how the modules are loaded.   Here are some of the changes:

* Updated the package.json deps
* Used import where I could, even in the web-worker
* Documented how to `npm link` to do local dev on this, and also how to make it work with Create React App
* Minor other refactorings.

### CODE REVIEW

File: src/utils/exr-parser.worker.js [hunk 1]
```diff
@@ -17,102 +17,104 @@
...
         if (exrImage) {
-            exrImage.delete();
+            exrImage["delete"]();
         }
         console.timeEnd('Decoding EXR'); // tslint:disable-line
     }
-};
+}
+;
+
```
Comment (engineersamuel) [inline src/utils/exr-parser.worker.js:120]: Note I had rewritten this file as a typescript file, then copied the js output back to this file, so there are some things changed here we may want to change back, like const -> var, it should be const.  Just noting that.

File: src/utils/exr-parser.worker.js [hunk 2]
```diff
@@ -17,102 +17,104 @@
...
-    } else {
+    }
+    else {
         handleJob(event.data);
     }
 });
 
-
-const exrwrapPath = require('file-loader?name=exr-wrap.js!../exr-wrap/exr-wrap.js');
-const exrwrapWasmPath = require('file-loader?name=exr-wrap.wasm!../exr-wrap/exr-wrap.wasm');
```
Comment (engineersamuel) [inline src/utils/exr-parser.worker.js:34]: This part doesn't just work when compiling and importing this as an npm module, this load the wasm from the file-loader, but then the exr-wrap.js will load it in a web worker env from the browser and still attempt to resolve as an XMLHttpRequest.  So I realize the intent here is to get Webpack to copy the wasm file to the built output, but that's already happening in the package.json scripts.

File: src/utils/exr-parser.worker.js [hunk 3]
```diff
@@ -17,102 +17,104 @@
  *  };
  */
 
-let openEXRLoaded = false;
-let queuedJobs = [];
-let OpenEXR;
+exports.__esModule = true;
 
-self.addEventListener('message', (event) => {
+var EXR = require("../exr-wrap/exr-wrap.js");
```
Comment (engineersamuel) [inline src/utils/exr-parser.worker.js:22]: This is an important change as previously the EXR was being bound as a global var I believe, much better to require it as a module like this.

Comment (cbreak-black) [reply src/utils/exr-parser.worker.js:22]: I faintly remember having problems with asynchronous loading / compiling of the webasm code. These variables were used to detect whether the wasm is already loaded and available or not, and to queue up decode requests if it's not available.

Comment (engineersamuel) [reply src/utils/exr-parser.worker.js:22]: Closing this and moving convo to https://github.com/disneyresearch/jeri/pull/17/files. But in general, I think this should just work now with the require like this.

**Comment by engineersamuel** [comment]
Closing in favor of #17

**Pull request closed without merging**